testing/trino-server-dev/etc/config.properties

```diff
@@ -32,6 +32,7 @@ plugin.bundles=\
   ../../plugin/trino-delta-lake/pom.xml,\
   ../../plugin/trino-blackhole/pom.xml,\
   ../../plugin/trino-cassandra/pom.xml,\
+  ../../plugin/trino-scylladb/pom.xml,\
```

Comment:
There is no need to add this plugin since we have a query runner.

Reply:
but why it is added for cassandra than?